src/reseeding.rs

```diff
@@ -39,73 +40,125 @@ use {RngCore, SeedableRng, Error, ErrorKind};
 ///   `ReseedingRng` with the ISAAC RNG. That algorithm, although apparently
 ///   strong and with no known attack, does not come with any proof of security
 ///   and does not meet the current standards for a cryptographically secure
-///   PRNG. By reseeding it frequently (every 32 MiB) it seems safe to assume
+///   PRNG. By reseeding it frequently (every 32 kiB) it seems safe to assume
 ///   there is no attack that can operate on the tiny window between reseeds.
 ///
 /// # Error handling
 ///
 /// If reseeding fails, `try_fill_bytes` is the only `Rng` method to report it.
 /// For all other `Rng` methods, `ReseedingRng` will not panic but try to
-/// handle the error intelligently; if handling the source error fails these
+/// handle the error intelligently through some combination of retrying and
+/// delaying reseeding until later. If handling the source error fails these
 /// methods will continue generating data from the wrapped PRNG without
 /// reseeding.
-///
-/// It is usually best to use the infallible methods `next_u32`, `next_u64` and
-/// `fill_bytes` because they can make use of this error handling strategy.
-/// Use `try_fill_bytes` and possibly `try_reseed` if you want to handle
-/// reseeding errors explicitly.
```

Comment:
Why delete this doc?

Reply:
Because you changed `try_reseed` to have the same error handling, so no reason to prefer them.